### DISCUSSION TITLE
Testing an app that is configured with GitHub OAuth App

### DISCUSSION THREAD

**Discussion by dtthuynh** [cypress-io/cypress#22237] [Questions and Help]
We need to test a login flow that uses GitHub to login, but in order to do so, we need to configure a GH OAuth app where the callback URL changes depending on a dynamically created environment that the app is installed on. We resort to using Cypress to drive this configuration on the OAuth app... but is there a better way, programatically?

**Comment by AnthonyMadia** [comment]
Any update on this? @dtthuynh
